x/concentrated-liquidity/clmodule/module.go

```diff
@@ -155,12 +154,12 @@ func (am AppModule) GenerateGenesisState(simState *module.SimulationState, s *si
 
 func (am AppModule) Actions() []simtypes.Action {
 	return []simtypes.Action{
-		simtypes.NewMsgBasedAction("CreateConcentratedPool", am.keeper, simulation.RandomMsgCreateConcentratedPool),
```

Comment:
planning on working on simulator after incentives, therefore disabling rn so that it doesnot conflict with CL hooks

Reply:
Please link issue here when created so we can resolve

Reply:
yup added https://github.com/osmosis-labs/osmosis/issues/4753